supports/tools/docker/README.md

```diff
@@ -1,8 +1,53 @@
+# Run Hadoop cluster with SSM in docker containers
+
+There are two cluster types:
+* singlehost
+* multihost
+
+## Singlehost configuration
+
+* Hadoop + SSM in one container
+* SSM metastore as postgres container
+
+Command to build docker images in singlehost cluster mode (from project root dir)
+
+```shell
+build-image.sh singlehost
+```
+
+Command to start docker containers 
+
+```shell
+docker-compose -f ${project-dir}/supports/tools/docker/singlehost/docker-compose.yaml up -d
```

Comment:
nit: `docker-compose` is [deprecated](https://docs.docker.com/compose/migrate/), we can use `docker compose` instead

Reply:
how quickly the world is changing. fixed